features/api/ideas_workshop_ideas.feature

```diff
@@ -420,6 +417,223 @@ Feature:
     }
     """
 
+  Scenario: As a logged-in user I can add my idea only with a name
+    When I add "Content-Type" header equal to "application/json"
+    And I am logged as "martine.lindt@gmail.com"
+    And I send a "POST" request to "/api/ideas" with body:
+    """
+    {
+      "name": "Mon idée"
+    }
+    """
+    Then the response status code should be 201
+    And the response should be in JSON
+    And the JSON should be equal to:
+    """
+    {
+        "theme": null,
+        "category": null,
+        "needs": [],
+        "author": {
+            "uuid": "d4b1e7e1-ba18-42a9-ace9-316440b30fa7",
+            "first_name": "Martine",
+            "last_name": "Lindt"
+        },
+        "published_at": null,
+        "committee": null,
+        "status": "DRAFT",
+        "votes_count": {
+            "important": 0,
+            "feasible": 0,
+            "innovative": 0,
+            "total": 0,
+            "my_votes": []
+        },
+        "uuid": "@string@",
+        "author_category": "ADHERENT",
+        "description": null,
+        "created_at": "@string@.isDateTime()",
+        "name": "Mon idée",
+        "slug": "mon-idee",
+        "days_before_deadline": 20,
```

Comment:
could be tricky, method is based on the createdAt and based on the execution speed it could result with 21 days, better use @integer@ to avoid random test fail

Reply:
```suggestion
        "days_before_deadline": @integer@,
```